packages/hint-axe/src/_locales/en/messages.json

```
@@ -43,6 +43,10 @@
         "description": "Metadata name",
         "message": "Language"
     },
+    "learnMore": {
+        "description": "aXe documentation link text",
+        "message": "Learn more in aXe"
```

Axe has switched the capitalization they use in their documentation so we no longer capitalize the "X". However, the site we'll actually be linking to is Deque University so we should probably call that out too..
```suggestion
        "message": "Learn more about this axe rule at Deque University"
```